src/test/scala/is/hail/expr/ir/IRSuite.scala

```diff
@@ -120,4 +121,17 @@ class IRSuite extends TestNGSuite {
     assertEvalsTo(ArrayFilter(a, "x",
       ApplyComparisonOp(LT(TInt32()), Ref("x", TInt32()), I32(6))), FastIndexedSeq(3))
   }
+
+  @Test def testArrayRange() {
+    val range = { (start: Int, stop: Int, step: Int) => ArrayRange(I32(start), I32(stop), I32(step)) }
```

Comment:
def is clearer?

I also I increasingly think we need an IR interface layer that removes some of the biolerplate and makes simple examples easier to write.